### PR TITLE
Create lnx_privileged_user_creation.yml

### PR DESCRIPTION
**Pull request by zakibro**
Adding new use case for tracking of Creation of privileged user in linux

### CODE REVIEW

**Comment by nasbench** [comment]
Any specific log you for these events from?

**Comment by zakibro** [comment]
These are from rhel - /var/log/messages, I do believe that an equivalent of this in Ubuntu would be /var/log/syslog

File: rules/linux/builtin/lnx_privileged_user_creation.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+author: Pawel Mazur
+date: 2022/12/21
+tags:
+    - attack.t1136.001
+    - attack.t1098
+    - attack.persistence
+logsource:
+    product: linux
+detection:
+    selection_new_user|contains:
```
Comment (frack113) [inline rules/linux/builtin/lnx_privileged_user_creation.yml:23]:
```suggestion
    selection_new_user:
```
cf https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_specification.md#lists

File: rules/linux/builtin/lnx_privileged_user_creation.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        - 'new user'
+    selection_uids_gids|contains:
```
Comment (frack113) [inline rules/linux/builtin/lnx_privileged_user_creation.yml:25]:
```suggestion
    selection_uids_gids:
```
eg https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_specification.md#lists

**Review by frack113 — CHANGES_REQUESTED** [review]

**Review by frack113 — APPROVED** [review]

**Review by nasbench — APPROVED** [review]

**Pull request merged**